mongodbatlas/performance_advisor_test.go

```diff
@@ -0,0 +1,237 @@
+package mongodbatlas
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/go-test/deep"
+)
+
+func TestPerformanceAdvisor_GetNamespaces(t *testing.T) {
+	client, mux, teardown := setup()
+	defer teardown()
+
+	projectID := "ORG-ID"
+	processName := "test:27017"
+
+	mux.HandleFunc(fmt.Sprintf("/"+performanceAdvisorNamespacesPath, projectID, processName), func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, http.MethodGet)
+		fmt.Fprint(w, `{
+				  "namespaces" : [ {
+					"namespace" : "data.zips",
+					"type" : "COLLECTION"
+				  }, {
+					"namespace" : "data.stocks",
+					"type" : "COLLECTION"
+				  } ]
+		}`)
+	})
+
+	body := &NamespaceOptions{
```

Comment:
checking the Ops Manager PR I just saw this is still referenced as body in our tests, let's fix this